qemu/qemu_s390x_test.go

```diff
@@ -32,7 +32,7 @@ var (
 	deviceVFIOString               = "-device vfio-ccw,host=02:10.0,devno=" + DevNo
 	deviceSCSIControllerStr        = "-device virtio-scsi-ccw,id=foo,devno=" + DevNo
 	deviceSCSIControllerBusAddrStr = "-device virtio-scsi-ccw,id=foo,bus=pci.0,addr=00:04.0,iothread=iothread1,devno=" + DevNo
-	deviceBlockString              = "-device virtio-blk-ccw,drive=hd0,scsi=off,config-wce=off,devno=" + DevNo + ",share-rw=on -drive id=hd0,file=/var/lib/vm.img,aio=threads,format=qcow2,if=none,readonly"
+	deviceBlockString              = "-device virtio-blk-ccw,drive=hd0,scsi=off,config-wce=off,devno=" + DevNo + ",share-rw=on,serial=hd0 -drive id=hd0,file=/var/lib/vm.img,aio=threads,format=qcow2,if=none,readonly"
```

Comment:
@alicefr @Jakob-Naucke  does s390x support serial ?

Reply:
@devimc it should work. However, I don't have access to an s390x VM anymore to have a try. @Jakob-Naucke is out until December, so I'm ccing @jschintag

Reply:
Thanks @alicefr - good to hear from you again :smile:

Reply:
yeah - but now with a red hat on head? :smile:

Reply:
@jschintag I'll merge this PR, please feel free to raise an issue/PR if this patch breaks s390x

Reply:
thank you guys